src/objective/rank_objective.hpp

```diff
@@ -276,6 +332,9 @@ class LambdarankNDCG : public RankingObjective {
   double max_sigmoid_input_ = 50;
   /*! \brief Factor that covert score to bin in Sigmoid table */
   double sigmoid_table_idx_factor_;
+  /*! \brief Position bias factors */
+  mutable std::vector<label_t> pos_biases_;
+  double learning_rate_;
```

Comment:
Add a description of this member. And mark as `const`.

Reply:
I guess we cannot mark it as const since we assign its value from config.

Reply:
What about assign the value of `learning_rate_` in constructor? Just like `seed_`. It is OK to leave it without `const`. This is just a suggestion :)